examples/server/server.cpp

```diff
@@ -3815,20 +3815,24 @@ int main(int argc, char ** argv) {
     // Router
     //
 
-    // register static assets routes
-    if (!params.public_path.empty()) {
-        // Set the base directory for serving static files
-        bool is_found = svr->set_mount_point("/", params.public_path);
-        if (!is_found) {
-            LOG_ERR("%s: static assets path not found: %s\n", __func__, params.public_path.c_str());
-            return 1;
-        }
+    if (!params.webui) {
+        LOG_INF("Web UI is %s\n", params.webui ? "enabled" : "disabled");
```

Comment:
```suggestion
        LOG_INF("Web UI is disabled\n");
```

Avoid dead code. Some compilers may throw warning on that.

Reply:
Thank you @ngxson